SELF_HOSTED_INSTRUCTIONS.md

```diff
@@ -0,0 +1,86 @@
+To create this folder structure run:
+
+```
+mkdir -p uwazi/log uwazi/uploaded_documents uwazi/custom_uploads uwazi/temporal_files
+.
+├── /home/user/uwazi
+│   ├── log/
+│   ├── uploaded_documents/
+│   ├── custom_uploads/
+│   ├── temporal_files/
+```
+
+- From within the `uwazi` folder, download the latest release .tar file:
+
+`wget https://github.com/huridocs/uwazi/releases/latest/download/uwazi.tgz`
+
+- Unpack the file. This will create a `prod` folder with the latest Uwazi production code:
+
+`tar xvf uwazi.tgz`
+
+- Rename to the `prod` folder to `lastest` and delete the .tar file:
+
+`mv prod latest && rm uwazi.tgz`
+
+- you should have now
+
+```
+.
+├── uwazi
+│   ├── latest/
+│   ├── log/
+│   ├── uploaded_documents/
+│   ├── custom_uploads/
+│   ├── temporal_files/
+```
+
+- From within `uwazi/latest`, create a blank state with the desired database name. The platform deafults to `uwazi_development` if you don't define any name.
+
+`NODE_ENV=production yarn blank-state my_db_name`
+
+## Update Uwazi to a new release
+
+- Rename or delete previous `latest` folder
+- From within `uwazi` folder, download the latest release .tar file, unpack, rename to `latest` and remove the previous .tar file.
+
+```
+wget https://github.com/huridocs/uwazi/releases/latest/download/uwazi.tgz
+tar xvf uwazi.tgz
+mv prod latest && rm uwazi.tgz
+```
+
+- From within `uwazi/latest` run any migrations needed since your previous version and reindex the Elastic Search index:
+
+`NODE_ENV=production DATABASE_NAME=my_db_name INDEX_NAME=my_db_name yarn migrate-and-reindex`
+
+## Run Uwazi
+
+You should now be able to run an Uwazi instance in production mode:
+
+`DATABASE_NAME=my_db_name INDEX_NAME=my_db_name NODE_ENV=production FILES_ROOT_PATH=/home/user/uwazi/ node server.js`
+
+By default, Uwazi runs on `localhost` port 3000, so point your browser to http://localhost:3000 and authenticate yourself with the default username "admin" and password "change this password now".
+
+It is advisable to create your own system service configuration. Check out the user guide for [more configuration options](https://github.com/huridocs/uwazi/wiki/Install-Uwazi-on-your-server).
+
+## How to upgrade from a previous install.sh installation
+
+_Note_ This is only required if you have a previous installation made with the now deprecated `install.sh` script.
+
+Previous installations with `install.sh` created the `uploaded_documents` and other folders inside the build folder, which made updating the package a complex process. The current folder structure is more friendly for upgrading processes. In order to migrate to the new structure and use the release that we publish just follow the [First one-time setup](#first-one-time-setup) but instead of creating the folders, move them from the previous installation path:
+
+```
+mv old_uwazi_path/uwazi-production/uploaded_files /home/user/uwazi
```

Comment:
There is a naming inconsistency: here it's `uploaded_files`, in the previous instructions is `uploaded_docuemnts`.  Either one of them is wrong, the mv needs to rename the folder, or the previous naming also requires an extra ENV in order to point to that folder?